x-pack/test_serverless/api_integration/test_suites/common/spaces.ts

```diff
@@ -13,24 +13,206 @@ export default function ({ getService }: FtrProviderContext) {
+
+          // without a request body we would normally a 400 bad request if the endpoint was registered
+          svlCommonApi.assertApiNotFound(body, status);
+        });
+
+        it('#getShareableReferences', async () => {
+          const { body, status } = await supertest
+            .post('/api/spaces/_get_shareable_references')
+            .set(svlCommonApi.getCommonRequestHeader());
+
+          // without a request body we would normally a 400 bad request if the endpoint was registered
+          svlCommonApi.assertApiNotFound(body, status);
+        });
+
+        it('#disableLegacyUrlAliases', async () => {
+          const { body, status } = await supertest
+            .post('/api/spaces/_disable_legacy_url_aliases')
+            .set(svlCommonApi.getCommonRequestHeader());
+
+          // without a request body we would normally a 400 bad request if the endpoint was registered
+          svlCommonApi.assertApiNotFound(body, status);
+        });
+      });
+
+      describe('internal', () => {
+        it('#get requires internal header', async () => {
+          let body: any;
+          let status: number;
+
+          ({ body, status } = await supertest
+            .get('/api/spaces/space/default')
+            .set(svlCommonApi.getCommonRequestHeader()));
+          // expect a rejection because we're not using the internal header
+          expect(body).toEqual({
+            statusCode: 400,
+            error: 'Bad Request',
+            message: expect.stringContaining(
+              'method [get] exists but is not available with the current configuration'
+            ),
+          });
+          expect(status).toBe(400);
+
+          ({ body, status } = await supertest
+            .get('/api/spaces/space/default')
+            .set(svlCommonApi.getInternalRequestHeader()));
+          // expect succes because we're using the internal header
+          expect(body).toEqual(
+            expect.objectContaining({
+              id: 'default',
+            })
+          );
+          expect(status).toBe(200);
+        });
+
+        it('#getAll requires internal header', async () => {
+          let body: any;
+          let status: number;
+
+          ({ body, status } = await supertest
+            .get('/api/spaces/space')
+            .set(svlCommonApi.getCommonRequestHeader()));
+          // expect a rejection because we're not using the internal header
+          expect(body).toEqual({
+            statusCode: 400,
+            error: 'Bad Request',
+            message: expect.stringContaining(
+              'method [get] exists but is not available with the current configuration'
+            ),
+          });
+          expect(status).toBe(400);
+
+          ({ body, status } = await supertest
+            .get('/api/spaces/space')
+            .set(svlCommonApi.getInternalRequestHeader()));
+          // expect succes because we're using the internal header
```

Comment:
__typo:__ (here and in a few other places in this file)
```suggestion
          // expect success because we're using the internal header
```